vardict.pl

```diff
@@ -2110,6 +2119,7 @@ sub toVars {
 	if ( $vars{ $p }->{ VAR } ) {
 	    for(my $vi = 0; $vi < @{ $vars{ $p }->{ VAR } }; $vi++) {
 		my $vref = $vars{ $p }->{ VAR }->[$vi];
+		my $genotype1current = $genotype1;
```

Comment:
I propose that we don't have to update the initial $genotype1 field, but have to create new variable $genotype1current, that will be updated only inside iteration for current variant. If it is incorrect and we have to combine genotype1 from all sequences with matcher &([ATGC]+), then maybe sort of variants will be enough.